pkg/coscheduling/README.md

```diff
@@ -14,11 +14,20 @@ sorting](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep/2-
 
 ## Tutorial
 ### PodGroup
-We use a special label named pod-group.scheduling.sigs.k8s.io/name to define a PodGroup. Pods that set this label and use the same value belong to the same PodGroup. 
+We use a special label named `pod-group.scheduling.sigs.k8s.io` to define a PodGroup. Pods that set this label and use the same value belong to the same PodGroup.
 ```
+# PodGroup CRD spec
+apiVersion: scheduling.sigs.k8s.io/v1alpha1
+kind: PodGroup
+metadata:
+  name: nginx
+spec:
+  scheduleTimeoutSeconds: 10
+  minMember: 3
+---
+# Add a label `pod-group.scheduling.sigs.k8s.io` to mark the pod belongs to a group
 labels:
-     pod-group.scheduling.sigs.k8s.io/name: nginx
-     pod-group.scheduling.sigs.k8s.io/min-available: "2"
+  pod-group.scheduling.sigs.k8s.io: nginx
```

Comment:
I filed https://github.com/kubernetes-sigs/scheduler-plugins/issues/440 because this label doesn't look right.